swagger/v3/component_schemas/ECFParticipantAttributes.yml

```diff
@@ -4,6 +4,7 @@ required:
   - full_name
   - updated_at
   - ecf_enrolments
+  - participant_id_changes
```

Comment:
could you add a new object participant id changes with the fields/explanations/examples to the spec please?